arcade/standard/blockglitch_4/map.xml

```diff
@@ -2,7 +2,8 @@
 <!-- #region meta -->
 <name>BlockGlitch 4</name>
 <slug>blockglitch_4_hell_edition</slug>
-<version>1.1.1</version>
+<!-- Commit ID: be9dd6ac464c4358640f7fa4903452abb4c19ae6 -->
```

Comment:
Is this line intentional?

Reply:
This line is intentional ballast of not very large significance (read as "build metadata").

The resulting `map.xml` thrown over the wall is generated using external templating tools not bundled with the map for security reasons. (I don't think this repository should contain executable files – however, using external templating tools reduces the amount of XML lines I have to work with to about a half) The commit ID it refers to is the commit SHA in an internal repository used to build the `map.xml`.